library/classes/X12837P.php

```diff
@@ -0,0 +1,1239 @@
+<?php
+/* X12837P Class
+ * 
+ * This program creates a 5010 837P file
+ *
+ * @package OpenEMR
+ * @author Rod Roark <rod@sunsetsystems.com>
+ * @author Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2009 Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2018 Stephen Waite <stephen.waite@cmsvt.com>
+ * @link https://github.com/openemr/openemr/tree/master
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once(dirname(__FILE__) . "/../invoice_summary.inc.php");
+
```

Comment:
Place following line here:
namespace OpenEMR\Billing;

Reply:
btw, make sure there is an empty line before and after the above `namespace` line